platforms/blabsy-w3ds-auth-api/src/web3adapter/watchers/firestoreWatcher.ts

```
@@ -0,0 +1,199 @@
+                ? this.collection.path
+                : "collection group";
+        console.log(`Stopping watcher for collection: ${collectionPath}`);
+
+        if (this.unsubscribe) {
+            this.unsubscribe();
+            this.unsubscribe = null;
+            console.log(`Successfully stopped watcher for ${collectionPath}`);
+        }
+    }
+
+    private async handleError(error: any): Promise<void> {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)